apps/web/lib/getTeamMemberEmailFromCrm.ts

```diff
@@ -68,6 +68,14 @@ async function getAttributeRoutingConfig(
     },
     select: {
       routes: true,
+      responses: {
```

Comment:
I think we could rewrite this query to be something like this which is simpler and must be faster

```
const response = await prisma.app_RoutingForms_Form.findUnique({where:{
id: routingFormResponseId,
include: {
form: {
select: {routes:true}
}
}
}})
```